forge.config.js

```diff
@@ -100,6 +100,8 @@ const config = {
     [
       '@electron-forge/plugin-webpack',
       {
+        devContentSecurityPolicy:
+          "default-src 'self' 'unsafe-inline' data:; script-src 'self' 'unsafe-eval' 'unsafe-inline' data:; connect-src 'self' 'unsafe-inline' *.ingest.sentry.io data:;",
```

Comment:
**question**: will you explain this one to me?

Reply:
Before I added this, I was getting errors about CSP related to fetch requests to the sentry.io DNS.

```
"Refused to connect to 'https://o1150892.ingest.sentry.io/api/6225803/envelope/?sentry_key=***&sentry_version=7' because it violates the following Content Security Policy directive: "default-src 'self' 'unsafe-inline' data:". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.
```